neural_compressor/quantization.py

```diff
@@ -21,372 +21,14 @@
 import random
 import numpy as np
 from .config import _Config, options
-from .model.model import BaseModel, get_model_fwk_name, get_model_type, Model, MODELS
+from .data.dataloaders.dataloader import check_dataloader
```

Comment:
suggest to export those symbols at the top namespace.

for example, 

from .data import check_dataloader
from .metric import register_customer_metric